internal/services/media/media_transform_resource.go

```diff
@@ -96,6 +99,55 @@ func resourceMediaTransform() *pluginsdk.Resource {
 										Required:     true,
 										ValidateFunc: validation.StringInSlice(encodings.PossibleValuesForEncoderNamedPreset(), false),
 									},
+									"preset_configuration": {
```

Comment:
configuration seems redundant?
```suggestion
									"preset": {
```

Reply:
changed to `preset`